apps/kirjs/src/app/modules/svg-race/timer/timer.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'kirjs-timer',
+  templateUrl: './timer.component.html',
+  styleUrls: ['./timer.component.css']
+})
+export class TimerComponent implements OnInit {
```

Comment:
This component isn't used anywhere

Reply:
Going to be used it the game, though it should prob be made generic